example/ios/Supporting Files/gradle.properties

```diff
@@ -4,8 +4,8 @@ kotlin.code.style=official
 kotlin.mpp.stability.nowarn=true
 android.useAndroidX=true
 
-android_gradle_plugin_version=7.0.0-alpha15
-kotlin_version=1.4.32
+android_gradle_plugin_version=7.0.0
+kotlin_version=1.5.10
```

Comment:
1.5.10 is used because the compose plugin complains that stability was not checked for 1.5.20+. the check could be turned off, but I don't think we should

Reply:
and even if the check is suppressed the compose build fails with 

```
java.lang.InstantiationError: org.jetbrains.kotlin.ir.util.TypeTranslator
	at androidx.compose.compiler.plugins.kotlin.lower.AbstractComposeLowering.<init>(AbstractComposeLowering.kt:172)
	at androidx.compose.compiler.plugins.kotlin.lower.ClassStabilityTransformer.<init>(ClassStabilityTransformer.kt:61)
	at androidx.compose.compiler.plugins.kotlin.ComposeIrGenerationExtension.generate(ComposeIrGenerationExtension.kt:69)
```

for which I wasn't able to find a fix

Reply:
as agreed. 2 steps update will be performed